server/relaypeerservice.go

```diff
@@ -156,19 +156,19 @@ func (s *relayPeerService) handleSurveyReq(req surveyBroadCastRequest) {
 
 	foundStale = false
 
-	//broadcast request to all connected relays
+
 	s.relayMutex.RLock()
 	now := time.Now().Unix()
 	if relayList, ok := s.RelayMap[req.ou]; ok {
 		for relayuuid, robj := range relayList {
-			if now > robj.timeStamp && (now-robj.timeStamp) > int64(maxRelayIdle) { //5min max toleration
```

Comment:
Looks like this is removed by mistake. @Omkar0114  Can you check and revert back change?

Reply:
Also check below errors:
```
Error: server/relaypeerservice.go:186:2: expected declaration, found s
Error: server/relaypeerservice.go:406:2: expected declaration, found s
Error: server/relaypeerservice.go:451:2: expected declaration, found ctx
Error: server/relaypeerservice.go:537:2: expected declaration, found ctx
```

Reply:
I make the change revert in `server/relaypeerservice.go` and few errors I saw in the local copy. Let's check now